tequilapi/endpoints/service_test.go

```diff
@@ -74,15 +74,15 @@ func Test_AddRoutesForServiceAddsRoutes(t *testing.T) {
 	}{
 		{
 			http.MethodGet, "/services", "",
-			http.StatusOK, `[{"id":"11111111-9dad-11d1-80b4-00c04fd430c0","proposal":{"id":1,"providerId":"0xProviderId","serviceType":"testprotocol","serviceDefinition":{"locationOriginate":{"asn":"LT","country":"Lithuania","city":"Vilnius"}}},"status":"NotRunning","options":{"protocol":"","port":0}}]`,
+			http.StatusOK, `[{"id":"11111111-9dad-11d1-80b4-00c04fd430c0","proposal":{"id":1,"providerId":"0xProviderId","serviceType":"testprotocol","serviceDefinition":{"locationOriginate":{"asn":"LT","country":"Lithuania","city":"Vilnius"}}},"status":"NotRunning"}]`,
```

Comment:
Is `NotRunning` a status? If service is not running, it should return `404` in `GET /services/:id` and should not be included in `GET /services`

Reply:
`/services/:id` is meant to search for service instances, 404 seems logical to be related with authorisation of resource, should we link it with state of resource.
Service instance should be removed from service pool, then it stops.
Anyways it's a different scope